routeros/mikrotik_serialize.go

```diff
@@ -269,12 +269,18 @@ func MikrotikResourceDataToTerraform(item MikrotikItem, s map[string]*schema.Sch
 	var diags diag.Diagnostics
 	var err error
 	var transformSet map[string]string
+	var skipFields map[string]struct{}
 
 	// {"channel": "channel.config", "mikrotik-field-name": "schema-field-name"}
 	if ts, ok := s[MetaTransformSet]; ok {
 		transformSet = loadTransformSet(ts.Default.(string), false)
 	}
 
+	// "field_first", "field_second", "field_third"
```

Comment:
@vaerh this 3 lines are a bit structural change. heads up, double check if you think it makes sense.
What I am trying to do here is be able to skip fields from being passed to terraform as it throws warnings about fields being missing in the schema.

Reply:
It looks logical and good. In my opinion it should work without errors.